lms/djangoapps/ccx/tests/utils.py

```diff
@@ -77,6 +85,17 @@ def setUp(self):
 
         # Create instructor account
         self.coach = AdminFactory.create()
+
+        # adding staff to master course.
+        self.staff = UserFactory()
```

Comment:
@amir-qayyum-khan do you have permissions to update master? mitocw/master can always be updated to edx/master. I'm doing it now, but you should feel free to so it any time.